src/components/impactstudio/ActivityTimeline.vue

```diff
@@ -395,30 +415,12 @@ export default {
     }
   },
   created() {
-    const MILLISECONDS_PER_DAY = 1000 * 60 * 60 * 24
```

Comment:
@pauline2k Is this line a dupe of line 88?

Reply:
@lyttam Line 88 is the new location. The constant needed to be accessible to the whole component, not just one method.